server/bleep/src/semantic.rs

```diff
@@ -592,12 +592,11 @@ pub fn deduplicate_with_mmr(
 pub fn deduplicate_snippets(
     mut all_snippets: Vec<Payload>,
     query_embedding: Embedding,
+    output_count: usize,
```

Comment:
We can just call this `k` and remove the assignment

Reply:
No, I'd like this to be a human-readable variable name at least in high-level APIs.

Reply:
`k` is a conventional name in search. It's common to talk about the `top-k` elements in a ranked list for example.